### PR TITLE
:rocket: fix the well-known assets

### PR DESCRIPTION
**Pull request by jlandure**

### CODE REVIEW

File: firebase.json [hunk 1]
```diff
@@ -6,6 +6,16 @@
     "public": "build/unbundled",
     "rewrites": [
       {
+        "source": "/.well-known/assetlinks.json",
+        "headers" : [
```
Comment (cyrilmottier) [inline firebase.json]: Formatting is KO (no space needed pre colon)

File: firebase.json [hunk 2]
```diff
@@ -6,6 +6,16 @@
...
+          {
+            "key" : "Content-Type",
```
Comment (cyrilmottier) [inline firebase.json]: Formatting is KO (no space needed pre colon)

File: firebase.json [hunk 3]
```diff
@@ -6,6 +6,16 @@
...
+            "value" : "application/json"
```
Comment (cyrilmottier) [inline firebase.json]: Formatting is KO (no space needed pre colon)

File: firebase.json [hunk 4]
```diff
@@ -6,6 +6,16 @@
...
+          }
+        ],
+        "type" : 200
```
Comment (cyrilmottier) [inline firebase.json]: Formatting is KO (no space needed pre colon)

File: firebase.json (hunk 1 above)
Comment (cyrilmottier) [reply firebase.json]: It looks like the file is alpha ordered. Put `headers` before `source`?

File: polymer.json [hunk 5]
```diff
@@ -21,7 +21,8 @@
     "bower.json",
     "manifest.json",
     "robots.txt",
-    "sitemap.xml"
+    "sitemap.xml",
+    ".well-known/**"
```
Comment (cyrilmottier) [inline polymer.json]: Looks like the list is alphaordered. Generally `.` is before all letters. I would put it at the beginning.

File: firebase.json (hunk 1 above)
Comment (jlandure) [reply firebase.json]: I prefer to keep `source` in top.

Comment (cyrilmottier) [reply firebase.json]: I can understand it 👍

**Comment by cyrilmottier** [comment]
Everything is OK for me.

**Review by cyrilmottier — APPROVED** [review]

**Pull request merged**